### PR TITLE
Husking can be fixed with plastic surgery now

### PR DESCRIPTION
**Pull request by dragoslava**
More of a oversight but, husking happens with burn victims so that should of been fixed in the initial PR

### CODE REVIEW

**Comment by dragoslava** [comment]
!review

**Comment by ParadoxSpace** [comment]
How can plastic surgery fix a complete lack of fluids in the body.

**Comment by Pacmandevil** [comment]
a lot of Botox

**Comment by Arrow768** [comment]
> How can plastic surgery fix a complete lack of fluids in the body.

If you are husked due to ling, you are dead anyway. So that wont help you.
If you are husked due to severe burns and are still alive, you apparently sill have fluids in the body or you would be dead.

File: code/modules/surgery/facial_surgery.dm [hunk 1]
```diff
@@ -68,8 +68,9 @@
 
 	end_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
 		var/obj/item/organ/external/head/head = target.get_organ(target_zone)
-		if(head.disfigured)
+		if(head.disfigured || HUSK in target.mutations)
```
Comment (skull132) [inline code/modules/surgery/facial_surgery.dm]:
```suggestion
		if(head.disfigured || (HUSK in target.mutations))
```

**Pull request merged**